package.json

```diff
@@ -2,13 +2,13 @@
   "name": "electron-react",
   "version": "1.0.0",
   "description": "",
-  "main": "main.ts",
-
+  "main": "main.js",
+  "type": "commonjs",
```

Comment:
@alvma1945 nervous about this. I'm pretty sure I had to remove this to get the back end to work...